test/fixtures/transaction-response.json

```diff
@@ -0,0 +1,89 @@
+{
+  "type": "payment",
+  "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
+  "specification": {
+    "source": {
+      "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
+      "amount": {
+        "currency": "XRP",
+        "value": "1.112209"
```

Comment:
I'm trying to use the convention that if something doesn't apply, then it is not present. The places where empty string still appears are because that is coming from ripple-lib-transactionparser. At some point either that module will be changed or an adapter will be used to get rid of the counterparty fields.